scripts/parse_new_build_warnings.py

```diff
@@ -165,11 +165,16 @@ def parse_new_build_warnings_from_directory(
 
 def export_build_warnings(warnings_dict: Dict[str, Set[str]], output: str):
     with open(output, "w") as f:
+        f.write(
+            f"# New build warnings\nA List of all additional build warnings present at this hash\n"
+        )
         for target, warnings in warnings_dict.items():
-            f.write(f"New build warnings from target: {target}\n")
+            if not len(warnings):
+                continue
+            f.write(f"## {target}\n```")
             for warning in warnings:
                 f.write(warning)
```

Comment:
Just want to double check, each warning in the list ends with a `\n` right?

Reply:
Yes each warning in the list ends with a \n